propulate/propulator.py

```diff
@@ -18,11 +19,9 @@
 
 class Propulator:
```

Comment:
add methods like _send_emigrants that only raise NotImplementedError as abstract base for the island adapted propulators